src/illuminant.rs

```diff
@@ -333,8 +333,8 @@ impl Illuminant {
     /// seperately to limit the size of the main web assembly library.
```

Comment:
Is this documentation outdated, or am I not understanding how these wasm bindings work? Where is it downloading extra color patch data? I think this part of the docs should be removed?